src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Negotiate/NegotiateStream_Http_Tests.cs

```diff
@@ -93,11 +108,33 @@ public static void NegotiateStream_Http_AmbientCredentials()
         }
     }
 
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
+#else
     [ConditionalFact(nameof(Windows_Authentication_Available), nameof(Root_Certificate_Installed))]
-    [ActiveIssue(851)]
```

Comment:
These NegotiateStream tests are now failing in OuterLoops because the removal of ActiveIssue(851) allows them to run.  But they cannot succeed without manual setup.  I agree #851 was not a good issue to use, so I created new issue 1265 asking to create a new ConditionalFact to detect whether the manual setup has been done.  I recommend putting back the line [ActiveIssue(1265)] so these don't get run by CI until we can make them conditional.